test/fixtures/group-basic.json

```diff
@@ -18,10 +18,10 @@
   "output": {
     "user_id": "user-id",
     "time": 1388534400000,
-    "user_properties": {
+    "groups": {
```

Comment:
Relying on JSON serialization order in tests seems super sketchy. Unfortunately it looks like we don't have the test infrastructure to add some form of `decode` to a request body before matching so we could deserialize the body into a JSON object (like we have for query strings). Might be a good thing to add to the TODO list